### ISSUE TITLE
https://sbft24.github.io/tools/fuzzing Updates

### ISSUE THREAD

**Issue by vrthra** [sbft24/sbft24.github.io#36]
### tl;dr (Current)

 * Express your interest [here](https://forms.gle/bzbro1zAXoCxLBHd8) (Deadline: AoE Friday 20 Jan 2023).
 * Write a fuzzer for C/C++ programs (or choose an existing fuzzer and make your own novel modifications).
 * Maximize the number of bugs found and coverage achieved in 23 hours.
 * Integrate fuzzer into [FuzzBench](https://google.github.io/fuzzbench/getting-started/adding-a-new-fuzzer/) (Deadline: AoE Friday 17 Feb 2023).
 * Competition results will be reported live at SBST’23.

This should be changed to

### tl;dr (Changed)

 * Express your interest [here](https://forms.gle/pwxtqgEaN724NESC9) (Deadline: AoE 1 December 2023).
 * Write a fuzzer for C/C++ programs (or choose an existing fuzzer and make your own novel modifications).
 * Maximize the mutation score and coverage achieved in 23 hours.
 * Integrate fuzzer into [FuzzBench](https://google.github.io/fuzzbench/getting-started/adding-a-new-fuzzer/) (Deadline: AoE Friday 28 December 2023).
 * Competition results will be reported live at SBST’24.

Note that the form should be https://forms.gle/pwxtqgEaN724NESC9

### Timeline (Current)

 * 20 Jan’23: Registration deadline.
 * 23 Jan’23: Notification of acceptance.
 * 17 Feb’23: First Pull Request deadline (PR must be accepted at FuzzBench).
 * 24 Mar’23: Preliminary results on public benchmarks.
 * 03 Mar’23: Second Pull Request deadline.
 * 10 Mar’23: Communicate final evaluation results to participants.
 * 15 Mar’23: Camera-ready tool paper (4 pages, references included) deadline.
 * May’23: Official competition results and tool presentation live at SBFT workshop.

### Timeline (Changed)

* 01 Dec’23: Registration deadline.
 * 04 Dec'23: Notification of acceptance.
 * 28 Dec'23: First Pull Request deadline (PR must be accepted at FuzzBench).
 * 04 Jan'24: Preliminary results on public benchmarks.
 * 14 Jan'24: Second Pull Request deadline.
 * 21 Jan'24: Communicate final evaluation results to participants.
 * 25 Jan’24: Camera-ready tool paper (4 pages, references included) deadline.
 * 14 Apr’24: Official competition results and tool presentation live at SBFT workshop.

### Benchmarking Platform (Changed)
..... (keep the other stuff) ...

#### Related work
    [USENIX'23] "[Systematic Assessment of Fuzzers using Mutation Analysis](https://arxiv.org/abs/2212.03075)" Philipp Görz, Björn Mathis, Keno Hassler, Emre Güler, Thorsten Holz, Andreas Zeller, Rahul Gopinath
    [ESEC/FSE’21] “[FuzzBench: An Open Fuzzer Benchmarking Platform and Service](https://research.google/pubs/pub50600.pdf)”,
    J. Metzman, L. Szekeres, L.M.R. Simon, R.T. Sprabery, and A. Arya.

### Competition Process (Changes)

Registration by 1 December 2023

1.  Please prepare a short “tool report” (up to 2 pages in [IEEE conference format](https://www.ieee.org/conferences/publishing/templates.html); \documentclass[10pt,conference]{IEEEtran} without including the compsoc or compsocconf options) describing the technology and ideas behind the fuzzer you want to submit to the competition. The tool report should be sent to sbft24fuzzcomp@googlegroups.com as part of the registration. Please format your email subject as ‘[Report submission] {Fuzzer Name}’.
2. Please open a pull request (PR) by [forking FuzzBench](https://github.com/google/fuzzbench/fork) and [submitting a PR](https://github.com/google/fuzzbench/compare). We encourage you to start early and check your code with our CI tests, but you can edit your PR at any time before the pull request deadline (28 Dec 2023). Feel free to push the fuzzer implementation itself just right before the deadline. If there is any question regarding your PR, please let us know via sbft24fuzzcomp@googlegroups.com and format your email subject as ‘[PR Query] {Fuzzer Name}’.
3. Register for evaluation (see the Expression of interest link in TLDR). For registration questions, please let us know via sbft24fuzzcomp@googlegroups.com and format your email subject as ‘[Registration query] {Fuzzer Name}’.

Pull Request Deadline (on 28 Dec 2023)

Please remove the prices and awards.

### Organizers

The fuzzing competition is organized jointly by the [Software Engineering Group](https://soft-eng.sydney.edu.au/) of [University of Sydney](sydney.edu.au/) lead by [Rahul Gopinath](https://rahul.gopinath.org),  [Huaming Chen](https://www.sydney.edu.au/engineering/about/our-people/academic-staff/huaming-chen.html), and [Xi Wu](https://www.sydney.edu.au/engineering/about/our-people/academic-staff/xi-wu.html) , and [Philipp Görs](https://cispa.de/en/people/c01phgo), and [Joschua Schilling](https://cispa.de/en/people/c03josc) of [CISPA Helmholtz Center for Information Security, Germany](https://cispa.de)

**Comment by phi-go** [comment]
There is a typo with my name its: Görz

We should mention that we will try to offer access to the mutation testing integration with fuzzbench as soon as we have it completed.
Maybe we want to consider having a upper limit of inputs that will be checked, though, not sure about that.

Otherwise, looks good to me.

**Comment by checkdgt** [comment]
@vrthra + @phi-go ....I've updated the information from here and #35 ... please have a check and close this task when it looks good for you :)

**Comment by vrthra** [comment]
@checkdgt Could you also please update the relate work? We would like to remove the older ones and set the newer ones given in this issue.

**Comment by checkdgt** [comment]
@vrthra ... my bad, I thought that was appended and then realised there was a duplicate in there too. Have updated.
